core/tests/webdriverio_utils/ExplorationEditorTranslationTab.js

```diff
@@ -0,0 +1,554 @@
+  var closeAudioUploaderModalButton = $('.e2e-test-close-audio-upload-modal');
+  var audioUploadContainerElement = $('.e2e-test-audio-upload-container');
+  var activeTranslationTabElement = $('.e2e-test-active-translation-tab');
+  var translationFeedback = function(index) {
+    return $('.e2e-test-feedback-' + index);
+  };
+  var translationFeedbackText = function(index) {
+    return $('.e2e-test-feedback-' + index + '-text');
+  };
+  var translationHint = function(index) {
+    return $('.e2e-test-hint-' + index);
+  };
+  var translationHintText = function(index) {
+    return $('.e2e-test-hint-' + index + '-text');
+  };
+  var _selectLanguage = async function(language) {
+    await waitFor.visibilityOf(
+      languageSelectorElement,
+      'Language selector takes too long to appear.');
+    await languageSelectorElement.selectByVisibleText(language);
+  };
+  var stateNodeLabel = function(nodeElement) {
+    return nodeElement.$('.e2e-test-node-label');
+  };
+
+  this.deleteAudioRecord = async function() {
+    await action.click('Delete record button', deleteRecordButton);
+    await action.click(
+      'Confirm delete record button',
+      confirmDeleteRecordButton);
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.uploadAudioRecord = async function(audioPath) {
+    // Two such elements are in the DOM, but only the second is visible.
+    var uploadAudioButton = await uploadAudioButtonSelector();
+    var lastElement = uploadAudioButton.length - 1;
+    await action.click('Audio Record Button', uploadAudioButton[lastElement]);
+    var absPath = path.resolve(__dirname, audioPath);
+    await action.setValue(
+      'Audio upload input', audioUploadInput, absPath, false);
+  };
+
+  this.saveAudioRecord = async function() {
+    await action.click('Save uploaded audio button', saveUploadedAudioButton);
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.addAudioRecord = async function() {
+    await action.click('Start record button', startRecordButton);
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.stopAudioRecord = async function() {
+    await action.click('Stop record button', stopRecordButton);
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.confirmAudioRecord = async function() {
+    await action.click('Confirm record button', confirmRecordButton);
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.playAudioRecord = async function() {
+    await action.click('Play record button', playRecordButton);
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.uploadAudioFileForLanguage = async function(
+      language, relativePathOfAudioToUpload) {
+    await this.changeLanguage(language);
+    await this.openUploadAudioModal();
+    await this.uploadAudio(relativePathOfAudioToUpload);
+  };
+
+  this.setTranslation = async function(richTextInstructions) {
+    await action.click('Edit translation button', editTranslationButtton);
+    // eslint-disable-next-line oppia/e2e-practices
```

Comment:
In protractor code we have `element(by.tagName('state-translation-editor'))` and lint checks passes because it only checks for the pattern element(by.css) but in webdriverio every selector are in the form `$()` and hence lint checks looks for this pattern only so it fails here as we are using tag name instead of class name hence we need to disable lint check for this line.